thirdparty build fails because of olog/jersey

Since about March 16, thirdparty build fails for me with this error:

```
[INFO] Resolving dependencies of MavenProject: org.csstudio:edu.msu.nscl.olog.api:2.2.8-SNAPSHOT @ /usr/local/hudson/config/jobs/CSS_2_cs-studio-thirdparty/workspace/plugins/edu.msu.nscl.olog.api/pom.xml
[INFO] {osgi.os=linux, osgi.ws=gtk, org.eclipse.update.install.features=true, osgi.arch=x86_64}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: edu.msu.nscl.olog.api 2.2.8.qualifier
[ERROR]   Missing requirement: edu.msu.nscl.olog.api 2.2.8.qualifier requires 'bundle com.sun.jersey.bundle 1.19.0' but it could not be found
```

--- a/pom.xml
+++ b/pom.xml
@@ -72,7 +72,13 @@
         </repository>
       </repositories>
     </profile>
-    <!-- If a local repository is specified then enable that repository -->
+    <!-- If a local repository is specified then enable that repository
+         
+         This option is DEPRECATED; use csstudio.composite.repo:
+         Composite repo can list local or remote sites,
+         so it covers the 'local' case, but avoids
+         copying all artifacts plus dependencies into a csstudio.local.repo
+      -->
     <profile>
       <id>csstudio-local-repo-enable</id>
       <activation>
@@ -84,6 +90,22 @@
         <repository>
           <id>csstudio-local-repo</id>
           <url>file:${csstudio.local.repo}</url>
+          <layout>p2</layout>
+        </repository>
+      </repositories>
+    </profile>
+    <!-- If composite repo is defined, use it. -->
+    <profile>
+      <id>csstudio-composite-repo-enable</id>
+      <activation>
+        <property>
+          <name>csstudio.composite.repo</name>
+        </property>
+      </activation>
+      <repositories>
+        <repository>
+          <id>csstudio-composite-repo</id>
+          <url>file:${csstudio.composite.repo}</url>
           <layout>p2</layout>
         </repository>
       </repositories>

